src/Microsoft.VisualStudio.AppDesigner/ApplicationDesigner/DesignerTabButton.vb

```diff
@@ -280,6 +280,20 @@ Namespace Microsoft.VisualStudio.Editors.ApplicationDesigner
                 _button.PerformClick()
             End Sub
 
+            ''' <summary>
+            ''' Calls default action when select is called. Checks for correct flag
+            ''' </summary>
+            Public Overrides Sub [Select](flags As AccessibleSelection)
+                '''AccessibleSelection.TakeSelection is the flag set when an object is selected,
+                '''specfically from SelectionItem.Select If this flag is set, we call the default
+                '''action on the PageTab, which gives the desired behavior of selecting the item.
+                If (flags And AccessibleSelection.TakeSelection) = AccessibleSelection.TakeSelection Then
```

Comment:
It would probably be good to determine that only respecting `TakeSelection` is the correct behavior.

My own intuition says it is, but we don't want to accidentally miss some edge accessibility scenario where some different behavior is expected for `AddSelection`, for example.